Fix RTL issues

unho opened the pull request:
For issue #4848

SafaAlfulaij commented on pootle/static/css/style.css:
```diff
@@ -1160,6 +1166,15 @@ table.stats td.last-updated
+html[dir="rtl"] th.stats-number,
+html[dir="rtl"] td.stats-number,
+html[dir="rtl"] td.stats-percentage,
+html[dir="rtl"] table.stats td.last-updated
```
I think it's better that "table.stats td.last-updated" is aligned to right, beacuse that column contains RTL text ..

unho replied:
It is aligned exactly the opposite of non-rtl. Perhaps I am misunderstanding you?

![ltr](https://cloud.githubusercontent.com/assets/939880/16868097/0b241676-4a76-11e6-9e9f-4905fc07fc29.png)
![rtl](https://cloud.githubusercontent.com/assets/939880/16868098/0b2ac70a-4a76-11e6-96b1-9e36c47d043b.png)

unho replied:
This is now sorted out.

SafaAlfulaij commented:
More fixing (sorry :P):
```
html[dir="rtl"] @media screen and (max-width: 1440px) .path-summary-more .summary-1-col
{
    margin: 0 0 0 6.25%;
}

html[dir="rtl"] @media screen and (max-width: 1440px) .path-summary-more .summary-2-col
{
    margin: 0 6.25% 0 0;
}

html[dir="rtl"] .path-summary-more .top-scorers-table tr td.number:last-child
{
    text-align: left;
}

html[dir="rtl"] .check-count
{
    text-align: left;
}

html[dir="rtl"] .last-action
```

Current attribute changes in "html[dir="rtl"] .check-count" should be removed and only apply an alignment change.
"html[dir="rtl"] .last-action" should be removed as it doesn't apply anymore (Old stuff from 2.5?), and it adds some unneeded space.

unho commented:
@SafaAlfulaij Forgot to say that there is a staging server for this PR http://staging.translatehouse.org:8002/ Thanks for the suggestions.

unho commented:
@SafaAlfulaij Created and account safa/safa for you in http://staging.translatehouse.org:8002/ and updated with your suggestions.

dwaynebailey commented:
Did a quick review.  Some things we might need to look at, but not blockers for this PR:

* Numbers in columns are left aligned, is that right? Or is it just my LTR head
* This alignment causes some issues are they don't make it easy to see what is which column in the browse table
* Expaned navbar view: the icon for hte user appears far left so you actually get the page scrolled left just for the icon
* I'm concerned the icon, name order in the collapsed navbar is not correct
* Score is N+ in LTR its +N.  Does that work in RTL?  Should '+%d' be localised?
* Expanded view.  Last update user icon overlaps with the link to what was translated

SafaAlfulaij commented:
> Numbers in columns are left aligned, is that right? Or is it just my LTR head

Maybe you are correct. 2.5 was displaying them right-aligned.
> This alignment causes some issues are they don't make it easy to see what is which column in the browse table

If we're going to correct the numbers, then it's safe to make them all right-aligned.
> Expaned navbar view: the icon for hte user appears far left so you actually get the page scrolled left just for the icon

Thanks for the notice! I tried but didn't know how to fix it. :(
> I'm concerned the icon, name order in the collapsed navbar is not correct

Seems like the one above..
> Score is N+ in LTR its +N. Does that work in RTL? Should '+%d' be localised?

I would say adding:
```
html[dir="rtl"] .path-summary-more .top-scorers-table tr td.number:last-child {dir="rtl"}
```
> Expanded view. Last update user icon overlaps with the link to what was translated

That's because the action text isn't translated, this bug: #4121


Now there are these:
![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877181/a6abdb2c-4aaf-11e6-965f-f3549ce730b0.png)

The switch "button" to the left is really sticked with the "Suggest" button to the right. There should be some small space. Didn't know how to fix it. :(

![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877182/a6ae3606-4aaf-11e6-9a4d-ad1aedcbb631.png)

There is some more space than needed to the right of the English text.
For a reason, Chromium is flagging `padding-right=auto` attrib of `"tml[dir="rtl"] .suggestion-original` as invalid. Removing the `padding-right` attrib from the ltr selector makes it right. Strange that it's using the float from the rtl selector not the ltr one.

![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877541/517ef3f8-4ab1-11e6-9bc3-53a3fd904b22.png)

The arrows should be mirrored/flipped. Both of them have the same css and that's beyond my knowledge :smile:
This is for both the simplified and detailed one.

The last thing (for now?) is that the detailed statistics page need to be RTL (main html doesn't have `dir="rtl"` attrib set.

unho commented:
>> - Numbers in columns are left aligned, is that right? Or is it just my LTR head
>
> Maybe you are correct. 2.5 was displaying them right-aligned.

This is just the opposite of what we have for LTR.


>> - This alignment causes some issues are they don't make it easy to see what is which column in the browse table
>
> If we're going to correct the numbers, then it's safe to make them all right-aligned.
>> - Expanded navbar view: the icon for the user appears far left so you actually get the page scrolled left just for the icon
>
> Thanks for the notice! I tried but didn't know how to fix it. :(

I didn't saw this. Can you please share a screenshot?

Perhaps related, I fixed the separation between top contributors when details are collapsed.

>> - I'm concerned the icon, name order in the collapsed navbar is not correct
>
> Seems like the one above..

Not sure what are you talking about either.


>> - Score is N+ in LTR its +N. Does that work in RTL? Should '+%d' be localised?
>
> I would say adding:
> ```
> html[dir="rtl"] .path-summary-more .top-scorers-table tr td.number:last-child {dir="rtl"}
> ```
>> - Expanded view. Last update user icon overlaps with the link to what was translated
>
> That's because the action text isn't translated, this bug: #4121

Exactly, with correct localization this is gone.


> Now there are these:
> ![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877181/a6abdb2c-4aaf-11e6-965f-f3549ce730b0.png)
>
> The switch "button" to the left is really sticked with the "Suggest" button to the right. There should be some small space. Didn't know how to fix it. :(

Fixed.

> ![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877182/a6ae3606-4aaf-11e6-9a4d-ad1aedcbb631.png)
>
> There is some more space than needed to the right of the English text.
> For a reason, Chromium is flagging `padding-right=auto` attrib of `"tml[dir="rtl"] .suggestion-original` as invalid. Removing the `padding-right` attrib from the ltr selector makes it right. Strange that it's using the float from the rtl selector not the ltr one.

Fixed.

> ![screenshot_ _](https://cloud.githubusercontent.com/assets/3509617/16877541/517ef3f8-4ab1-11e6-9bc3-53a3fd904b22.png)
>
> The arrows should be mirrored/flipped. Both of them have the same css and that's beyond my knowledge :smile:
> This is for both the simplified and detailed one.
>
> The last thing (for now?) is that the detailed statistics page need to be RTL (main html doesn't have `dir="rtl"` attrib set.

unho commented:
@SafaAlfulaij We are landing the changes in this PR and closing it. I will open a new PR to catch up with the remaining issues, or any new one you might have found. Some of the latest issues you reported are fixed, but other issues were not.

unho commented:
@SafaAlfulaij Landed the fixes in this PR. These fixes don't cover all the issues you reported, so created #4925 to track them. Feel free to comment or report more RTL problems there.
